scripts/README.txt

```diff
@@ -0,0 +1,118 @@
+## Flow_stats's scripts
```

Comment:
Italo, since it's markdown could you rename `scripts/README.txt` to `scripts/README.md`? That way text editors would pick up the correct file type and GH would rendered it correctly too.

Reply:
Thanks for the suggestion, @viniarck. fixed in cdfc63d.